docs/en/writing-running-appium/tutorial/swipe-tutorial.md

```diff
@@ -0,0 +1,41 @@
+## Scroll / Swipe actions tutorial
+
+### Android vs iOS UI differences
+
+Appium uses 'XCUITest' driver for iOS and 'UIAutomator2','Espresso' for
```

Comment:
I think this could be skipped, since it might require updates when the drivers list gets changed (or Appium 2 gets released, where drivers could be loaded dynamically) Lets only mention these drivers we are going to describe